pkg/controller/core/clusterqueue_controller_test.go

```diff
@@ -0,0 +1,96 @@
+package core
```

Comment:
You forgot to add Copyright ^_^

Reply:
Oops...
Thanks for mentioning this :)))